metagraph/src/annotation/row_diff_builder.cpp

```diff
@@ -331,38 +335,26 @@ rd_succ_bv_type route_at_forks(const graph::DeBruijnGraph &graph,
             exit(1);
         }
 
-        rd_succ = rd_succ_bv_type(std::move(rd_succ_bv));
-
+        rd_succ = std::make_shared<rd_succ_bv_type>(std::move(rd_succ_bv));
+        rd_succ->serialize(f);
+ 
     } else {
         logger->warn("No count vectors could be found in {}. The last outgoing"
                      " edges will be selected for assigning RowDiff successors",
                      count_vectors_dir);
+        rd_succ = last;
+        if(dynamic_cast<graph::DBGSuccinct const*>(&graph)) {
```

Comment:
I really don't like doing this. If we try to serialize `last`, what exactly is the issue with conversion? How much of a slowdown would it be? Alternatively, would it be an option to make some general bit_vector load and add class tags to serialized versions?